homework/index.html

```diff
@@ -1,23 +1,98 @@
+
+<select>
+  <option value="0">alumni</option>
+  <option value="1">angular</option>
+  ...
+</select>
+
+
+2) A left-hand column that displays basic information about the selected repository.
+HYF Repository (link) "url"
+Description "description"
+Forks "fork"
+Updated at "updated_at"
+
+3) A right-hand column that displays a list of contributors to the repository.
+Contributors (list) "contributors_url"
+(link)
+
+also:
+At start-up your application should display information about the first repository as displayed in the select element.
+
+When the user changes the selection, the information in the web page should be refreshed for the newly selected repository.
+
+You should be able to click on the repository name of the selected repository to open a new browser tab with the GitHub page for that repository.
+
+You should be able to click on a contributor to open a new browser tab with the GitHub page for that contributor.
+
+You should render network errors to the DOM (see Figure 2 below for an example). Do not use console.log as regular users will not see the console output. Use the predefined alert-error class from style.css to style your error.
+
+Your UI should be responsive. Try it with Chrome Developer Tools in the browser, using a mobile phone format and a tablet format, portrait and landscape. If necessary, you can also do this work in week 2.
+Use CSS media queries and Flexbox to make the UI responsive.
+
+HANDING IN:
+npm test
+Push your week1 branch to GitHub:
+  git push -u origin week1
+Create a pull request for your week1 branch.
+
+
+ISSUES: using 'Select a repository' with an empty string as option as the first 'option' as I cannot achieve point 2 of the Functional Requirements ('At start-up your application should display information about the first repository as displayed in the select element.') with 'change' event
+
+-->
+
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+    <meta name="theme-color" content="#000000" />
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="format-detection" content="telephone=no" />
+    <link rel="apple-touch-icon" href="hyf.png" />
+    <link rel="shortcut icon" type="image/png" href="hyf.png" />
+    <title>HYF-GITHUB</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet" />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+
+  <body>
+    <div id="root">
+      <h1>Hack Your Future Repositories</h1>
+      <div id="repoList">
+        <select id="select">
+          <option value="">
+            <--Choose a Repository-->
+          </option>
+        </select>
+      </div>
+      <div id="selected"></div>
+      <div id="contributors"></div>
+    </div>
```

Comment:
I think you technically weren't supposed to change this index file (as per the instructions), but I personally think it's ok to have starting content here.